core-blocks/latest-posts/edit.js

```diff
@@ -21,6 +20,7 @@ import {
 	withAPIData,
 } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+import { dateI18n, format, getSettings } from '@wordpress/date';
```

Comment:
This means we should probably add `wp-date`  as a dependency to `wp-core-blocks`.

Reply:
I added `wp-date` to `wp-block-library`